src/_includes/toc-and-content.html

```diff
@@ -0,0 +1,58 @@
+<div class="row">
+  <div class="toc col-md-3">
+    <div class="toc-container">
+      <div class="d-none d-sm-none d-md-block d-lg-block">
+        <h2>Table of Contents</h2>
+        {% for item in include.toc_data %}
+        <h3>{{ item.title }}</h3>
+        <ul>
+          {% for subsection_item in item.subsection_items %}
+          <li class="h5">
+            <a href="#{{ subsection_item.url }}">{{ subsection_item.title }}</a>
+          </li>
+          {% endfor %}
+        </ul>
+        {% endfor %}
+      </div>
+      <div id="mobile-toc-nav" class="d-block d-sm-block d-md-none d-lg-none">
+        <div class="mobile-control-group main-nav-icons toc-and-content-mobile-nav">
+          <button
+            id="mobile-toc-nav-button"
+            class="mobile-control toggle-menu"
+            aria-expanded="false"
+            aria-controls="navigation"
+            data-toggle="collapse"
+            data-target="#navigation-two"
+          >
+            <h2>Table of Contents</h2>
+          </button>
+        </div>
+        <nav id="navigation-two" class="main-navigation singlelevel auto-highlight no-icons nav collapse">
+          <ul id="nav_list" class="top-level-nav">
```

Comment:
Does this element need to be a `<ul>`? Wondering if a `<div>` would work here instead, since the inner items aren't `<li>` - which is what is expected inside `<ul>`

Reply:
Honestly no idea. This is magical CA Web Standard code territory. I can test it out.

Reply:
Is this how they do it??? Yikes